app/assets/javascripts/admin/bulk_product_update.js.coffee

```diff
@@ -242,32 +241,28 @@ angular.module("ofn.admin").controller "AdminProductEditCtrl", ($scope, $timeout
     $window.location = destination
 
   $scope.packProduct = (product) ->
```

Comment:
"pack" means "prepare fields to save in the database", for example 
* "extracts variant_unit_with_scale into variant_unit and variant_unit_scale"
* "extracts unit_value and unit_description from unit_value_with_description"

https://github.com/openfoodfoundation/openfoodnetwork/pull/12787/commits/dc65c4a3595fbe59e851057d5893654051753e91#diff-c4c442b187d42bdc1fde8c83fa0c1211a6bb222914f3a0f82b9604fdb812bb13R524-R529 👇